client/src/components/Itinerary.js

```diff
@@ -0,0 +1,86 @@
+import React, { Component } from 'react';
+import {getItineraries} from '../actions/itineraryActions';
+import {connect} from 'react-redux';
+// import Toggle from './Toggle';
+import Activities from './Activities';
+
+
+class Itineraries extends Component {
+    constructor() {
+      super();
+      this.state = {
+        Itineraries: [],
+        showActivities: ""
+      };
+     
+    }
+
+
+componentDidMount() {
+    console.log(this.props)
+    const cityLink = this.props.match.params.id
+          this.props.getItineraries(cityLink);
+          
+        }
+        
+toggle = (event) =>{
+    event.preventDefault();
+    // e.persist();
+    console.log(event.target.id);
+    this.state.showActivities === event.target.id ? 
```

Comment:
You should decide a consistent pattern for writing some of your operators just to help make things easier to follow.  If you find yourself straying from one you decide on, you could setup some harsh linting rules to enforce it.